django/db/backends/sqlite3/schema.py

```diff
@@ -331,6 +332,15 @@ def remove_field(self, model, field):
     def _alter_field(self, model, old_field, new_field, old_type, new_type,
                      old_db_params, new_db_params, strict=False):
         """Perform a "physical" (non-ManyToMany) field update."""
+        # If SQLite version is 3.25 or higher just alter table instead of remaking it for renaming of columns.
+        if sqlite_version >= (3, 25, 0):
```

Comment:
We usually define these a `cached_property`s on the `DatabaseWrapper` objects.

https://github.com/django/django/blob/0b1b1eac78362b3aff9b41e9af350bc1d0d7e41e/django/db/backends/mysql/base.py#L328-L339

Reply:
Alright, is it still alright to import the version number or do you want me to make a query like in your example?

Reply:
Using the version number should be fine!